src/main/groovy/com/bmuschko/gradle/docker/utils/RegistryAuthLocator.groovy

```diff
@@ -0,0 +1,293 @@
+package com.bmuschko.gradle.docker.utils
+
+import com.bmuschko.gradle.docker.DockerRegistryCredentials
+import com.fasterxml.jackson.databind.JsonNode
```

Comment:
Instead of using Jackson, can we change this to use `JsonSlurper` instead? I don't necessarily want to depend on that library explicitly.